### ISSUE TITLE
!important

### ISSUE THREAD

**Issue by GGAlanSmithee** [GGAlanSmithee/css-objectifier#1]
Figure out how to handle `!important` styles.

[CSS syntax guidelines](http://www.w3.org/TR/CSS21/cascade.html)
[CSS syntax guidelines easy read](http://www.w3schools.com/css/css_syntax.asp)

``` javascript
if `declaration.property` already exists for `selector`
    if old `declaration.value` is `!important`
        if new `declaration.value` is !important`
            override existing `declaration.value` for `selector`
        else
            keep existing `declaration.value` for `selector`
    else
        override existing `declaration.value` for `selector`
else
    add `declaration.property` with `declaration.value` on `selector`
```

**Comment by GGAlanSmithee** [comment]
This issue will be resolved in #2, using regex to find !important styles.

**Comment by GGAlanSmithee** [comment]
Closing this, since this is covered by the above-mentioned issue
